sql/core/src/test/java/test/org/apache/spark/sql/JavaBeanDeserializationSuite.java

```diff
@@ -210,6 +212,17 @@ private static Row createRecordSpark22000Row(Long index) {
     return new GenericRow(values);
   }
 
+  private static String timestampToString(Timestamp ts) {
```

Comment:
shall we reuse FractionTimestampFormatter?

Reply:
We could but if we introduce a bug to the formatter, we could catch it by this test.

Reply:
One more thing, if we use the same formatter for expected and actual results, checking current timestamps doesn't make so much sense. In that case, we could test a concrete timestamp.